app/mailers/notifier.rb

```diff
@@ -87,8 +87,8 @@ def statement_pdf
     @statement_pdf ||= StatementPdfFactory.instance(@statement)
   end
 
-  def send_nucore_mail(to, subject, template_name = nil)
-    mail(subject: subject, to: to, template_name: template_name)
+  def send_nucore_mail(to, subject, options = {})
```

Comment:
I'd lean toward keyword args (with `nil` defaults) rather than an options hash to be more self-documenting.